android/BOINC/app/src/main/AndroidManifest.xml

```diff
@@ -17,7 +17,6 @@
   along with BOINC.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-        package="edu.berkeley.boinc"
```

Comment:
Why did you remove this?

Reply:
Android studio suggest to remove it from here and add it on build.gradle. I didn't object to that suggestion.

Reply:
It did automated autogenerated suggestion